push-to-gcs.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Should we run integration test before pushing changes to GCS as well?

Reply:
Currently integration test are running against the published init actions. They cannot test local changes. We need to think about how to change this later.